sdk/storage/azure-storage-queue/README.md

```diff
@@ -173,7 +173,8 @@ queueClient.createWithResponse(metadata, null, Duration.ofSeconds(30), Context.N
 or
 
 ```Java
```

Comment:
@gapra-msft @rickle-msft we should update this to use embedme.

Reply:
There is no `ReademeSample.java` and no `embedme` comments

Reply:
@alzimmermsft - I have updated this to use embedme,  please review.